ability_item_usage_hoodwink.lua

```diff
@@ -1,6 +1,7 @@
 ----------------------------------------------------------------------------
 --	Ranked Matchmaking AI v1.3 New Structure
 --	Author: adamqqq		Email:adamqqq@163.com
+--	Update Author: Archer Wayne		Email:archerwayne1216@gmail.com
```

Comment:
You can use author directly to highlight your contribution.

Reply:
sure, I`ll update the authors.